package.json

```diff
@@ -72,7 +72,8 @@
     "pnpm": {
         "overrides": {
             "playwright": "1.45.0",
-            "typescript": "~4.9.5"
+            "typescript": "~4.9.5",
+            "@posthog/icons": "0.13.1"
```

Comment:
@mariusandra pnpm was being weird because most products list icons as a peer dependency and so we had several versions of the icons package listed in the pnpm lock file

that meant that the toolbar was picking up 2 copies of it at almost half a meg total

this seemed the only way to say "hallo everyone this version please" without needing to edit in all the package.jsons